README.md

```diff
@@ -224,24 +234,123 @@ Options passed to `start_link/2` as the options `{client_opts, [{connection_opts
 * `{batch_size, non_neg_integer()}`
 
   If commands are queued up in the process message queue, this is the maximum
-  number of messages that will be received and sent in one call.
+  number of messages that will be received and sent in one call. Default 16.
 
 * `{tcp_options, [gen_tcp:connect_option()]}`
 
   Options passed to `gen_tcp:connect/4`.
 
+* `{tcp_connect_timeout, timeout()}`
+
+  Timeout passed to gen_tcp:connect/4. Default infinity.
+
 * `{tls_options, [ssl:tls_client_option()]}`
 
   Options passed to `ssl:connect/3`. If this config parameter is present
   TLS will be used.
 
+* `{tls_connect_timeout, timeout()}`
+
+  Timeout passed to ssl:connect/3. Default infinity.
+
 * `{push_cb, push_cb()}`
 
   Callback for push notifications.
 
 * `{response_timeout, non_neg_integer()}`
 
-  Timeout when waiting for a response from Redis, in milliseconds.
+  Timeout when waiting for a response from Redis in milliseconds. Default 10000.
+
+  When a timeout happens, the connection is closed and the client attempts to
+  set up a new connection. See the client option `node_down_timeout` above.
+
+Info messages
+-------------
+
+When one or more pids have been provided as the option `{info_pid, [pid()]}` to
+`start_link/2`, these are the messages ered sends. All messages are maps with at
+least the key `msg_type`.
+
+Messages about the cluster as a whole:
+
+* `#{msg_type := cluster_ok}` is sent when the cluster is up and and runnning
+  and ered is connected to all nodes.
+
+* `#{msg_type := cluster_not_ok, reason := Reason}` is sent when something is
+  wrong, where Reason is one of the following:
+
+  * `master_down` if one of the master nodes is down or unreachable.
+
+  * `master_queue_full` if one of the master nodes is so busy that the maximum
+    number of queued commands for the node is reached and any further command to
+    that node would be rejected immediately.
+
+  * `pending` if ered has not yet established connections to all nodes.
+
+  * `too_few_nodes` if the cluster has fewer than two nodes.
+
+  * `not_all_slots_covered` if some cluster slot doesn't belong to any node in
+    the cluster.
+
+  * `too_few_replicas` if any of the master nodes has fewer replicas than the
+    minimum number as specified using the option `{min_replicas,
+    non_neg_integer()`. See options above.
+
+* `#{msg_type := slot_map_updated, slot_map := list(), map_version :=
+  non_neg_integer()}` is sent when the cluster slot-to-node mapping has been
+  updated.
+
+* `#{msg_type := cluster_slots_error_response, response := any()}` is sent when
+  there was an error updating the cluster slot-to-node mapping.
+
+Messages about the connection to a specific node are on the following form:
```

Comment:
```suggestion
Messages about the connection to a specific node are in the following form:
```

Reply:
Just a suggestion, felt more natural to me..but not sure

Reply:
I had to look it up. "In the form" seems to be the right phrasing. ("On" is probably my Swedish.)